src/client/app/utils/token.ts

```diff
@@ -10,6 +15,11 @@ export function getToken(): string {
 	return token;
 }
 
+/**
+ * Check if there has Token in local storage
+ *
+ * @returns {boolean} If has Token, return True; otherwise, catch the error
```

Comment:
Similarly to above and a couple of other small changes:
If there is a Token, return true; otherwise return false